src/tasks/index.ts

```diff
@@ -44,6 +45,11 @@ export const config: ConfigLoader.Config = {
 };
 
 export let worker: Worker | null = null;
+
+export function isPineTasksAvailable() {
```

Comment:
```suggestion
export function canExecuteTasks() {
```

Maybe personal preference, but I find this to be clearer naming